docs/source/configuration/overview.mdx

```diff
@@ -514,6 +514,42 @@ See [Tracing in the Apollo Router](./tracing/).
 
 ### TLS
 
+#### TLS termination
+
+Clients can connect to the router directly over HTTPS, without terminating TLS in an intermediary. It can be configured under the `tls` configuration section:
+
+```yaml
+tls:
+  supergraph:
+    certificate: ${file./path/to/certificate.pem}
+    certificate_chain: ${file./path/to/certificate_chain.pem}
+    key: ${file./path/to/key.pem}
+```
+
+The file referenced in the `certificate_chain` value is expected to be the combination of several PEM certificates, concatenated together into a single file (as is commonplace with Apache TLS configuration).
+
+Supported TLS versions:
```

Comment:
This may be better places in the generic TLS section.

Reply:
moved in d3e70e36